### PR TITLE
EKS POC additional work

### PR DESCRIPTION
**Pull request by yakloinsteak**
- **Compile additional assets**

https://github.com/greenriver/devops-requests/issues/889

## Description

Compile additional assets and EKS-related updates.

## Type of change
- [x] Bug fix

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: app/models/dj_metrics.rb [hunk 1]
```diff
@@ -65,14 +62,14 @@ def refresh_queue_sizes!
     Delayed::Job.where('failed_at IS NULL').where('locked_by IS NULL').group(:queue).count.each do |queue, size|
```
Comment (eanders) [inline app/models/dj_metrics.rb]: Is there a reason we wouldn't use:?
```suggestion
    Delayed::Job.where(failed_at: nil, locked_by: nil).group(:queue).count.each do |queue, size|
```

File: docker/app/docker-compose.yaml [hunk 2]
```diff
@@ -9,8 +9,8 @@ x-app: &app
     dockerfile: docker/app/Dockerfile
     args:
       BUILD_TAG: 3.1.6-alpine3.20
-      USER_ID: ${USER_ID:-1001}
-      GROUP_ID: ${GROUP_ID:-1001}
+      USER_ID: ${USER_ID:-1000}
+      GROUP_ID: ${GROUP_ID:-1000}
```
Comment (eanders) [inline docker/app/docker-compose.yaml]: Is this going to break some development environments?
If we need a specific default, maybe we should note why.

Comment (yakloinsteak) [reply docker/app/docker-compose.yaml]: I'll just change it back.

**Comment by yakloinsteak** [comment]
@eanders Could we merge this or I could deploy to ECS QA again since it's been a while.

**Comment by eanders** [comment]
@yakloinsteak looks good, merged.

**Pull request merged**